.github/workflows/oidc-test.yml

```diff
@@ -73,6 +72,17 @@ jobs:
         run: |
           jf rt s "some-repo/"
 
+      - name: Test outputs
+        shell: bash
+        run: |
+            echo "os is $RUNNER_OS"           
+            if [ -z "${{ steps.setup.outputs.jf_oidc_token }}" ]; then
+              exit 1
+            fi
+            if [ -z "${{ steps.setup.outputs.jf_oidc_user }}" ]; then
+              exit 1
+            fi
```

Comment:
```suggestion
      - name: Test User Output
        shell: bash
        run: test -n "${{ steps.setup-jfrog-cli.outputs.jf-oidc-user }}"

      - name: Test Token Output
        shell: bash
        run: test -n "${{ steps.setup-jfrog-cli.outputs.jf-oidc-token }}"
```

Reply:
that did not work so I reverted it to my working test